tests/notebooks/test_moving_between_models.py

```diff
@@ -0,0 +1,25 @@
+from testbook import testbook
+from dffml import IntegrationCLITestCase
+
+
+class TestNotebook(IntegrationCLITestCase):
```

Comment:
Let's rebase in master which got rid of `IntegrationCLITestCase`

```suggestion
class TestNotebook(AsyncTestCase):
```

Reply:
@pdxjohnny couldn't get testbook to work with Async. Used unnittest.TestCase.